applet/src/keyb.c

```diff
@@ -148,7 +157,7 @@ void handle_hotkey( int keycode )
             //mb_send_beep(beep_event_probe);
             break ;
         case 15 :
-            syslog_redraw();
+            bp_send_beep(BEEP_TEST_2);
```

Comment:
maybe a copy&paste error when moving key # back to it's original function, can't check now, have to look into my sources tonight.